static/js/containers/DashboardPage.js

```diff
@@ -550,6 +550,10 @@ class DashboardPage extends React.Component {
     );
   }
 
+  openLinkInNewTab = (url: ?string): void => {
+    window.open(url, '_blank');
+  }
```

Comment:
It seems unnecessary to create this method on `DashboardPage` and pass it down through the component tree - I think it would be better either to directly call `window.open` where we need it or make this a small util function. When testing it I think we should be able to use sinon to stub out `window.open` for testing purposes.